apps/publisher/themes/default/partials/lifecycle-state-duration.hbs

```diff
@@ -0,0 +1,23 @@
+<!--
```

Comment:
Thank you Sameera,
This entire code block has been refactored with new time in lifecycle feature.